pkg/engine/apply.go

```diff
@@ -80,7 +83,18 @@ func getLatest(target *Target) (plumbing.Hash, error) {
 		Force:           true,
 		InsecureSkipTLS: false,
 		CABundle:        []byte{},
-	}); err != nil && err != git.NoErrAlreadyUpToDate && !target.disconnected {
+	}
+	// if using ssh, change auth to use ssh key
+	if target.ssh {
+		logger.Infof("git clone %s ", target.url)
+		authValue, err := ssh.NewPublicKeysFromFile("git", target.sshKey, target.password)
+		if err != nil {
+			logger.Infof("generate publickeys failed: %s\n", err.Error())
```

Comment:
@djach7 the logger.Infof doesn't need the `\n`

Reply:
Updated